android/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,7 @@
   <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
   <uses-permission android:name="android.permission.USE_FULL_SCREEN_INTENT" />
   <uses-permission android:name="android.permission.SCHEDULE_EXACT_ALARM" />
+  <uses-permission android:name="android.permission.BROADCAST_CLOSE_SYSTEM_DIALOGS" />
```

Comment:
sorry I'm mobile right now but I think I read something about a change with this one in android 11 or 12, like maybe it wasn't allowed? I could be off - that is cmpletely crossed - with window overlay though, this just made my spidey-sense tingle

Reply:
ah, interesting. think it should be safe and E2E tests passed, but will double-check. Seems to have been deprecated for Android 12 https://developer.android.com/reference/android/content/Intent#ACTION_CLOSE_SYSTEM_DIALOGS

Reply:
😆 that is more than deprecated, this warning tells me it's not a usable solution - it shouldn't be touched anymore I think - for any app running on Android 12 it will be dropped no matter what and if you target and run on Android 12 it'll throw a SecurityException - ouch

> This constant was deprecated in API level 31.
> This intent is deprecated for third-party applications starting from Android Build.VERSION_CODES#S for security reasons. Unauthorized usage by applications will result in the broadcast intent being dropped for apps targeting API level less than Build.VERSION_CODES#S and in a SecurityException for apps targeting SDK level Build.VERSION_CODES#S or higher. Instrumentation initiated from the shell (eg. tests) is still able to use the intent. The platform will automatically collapse the proper system dialogs in the proper use-cases. For all others, the user is the one in control of closing dialogs.